src/components/form/form-button/form-button.spec.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import 'jest-styled-components';
+import { rootTagTest } from '../../../utils/helpers/tags/tags-specs/tags-specs';
+import FormButton from './form-button.component';
+import Button from '../../button';
+
+const renderFormButton = (props, render = shallow) => {
+  return render(
+    <FormButton
+      { ...props }
+    />
+  );
+};
+
+describe('FormButton', () => {
+  let wrapper, clickFunction, button;
+
+  describe.each(['save', 'cancel'])(
+    'the primary action buttons',
+    (name) => {
+      beforeEach(() => {
+        wrapper = renderFormButton({
+          formButtonName: name,
+          [`${name}ButtonProps`]: { className: 'my-custom-button-class' }
+        });
+      });
+
+      it('renders a custom class if provided', () => {
+        expect(wrapper.find('.my-custom-button-class').exists()).toBeTruthy();
+      });
+
+      it('renders the default text', () => {
+        button = wrapper.find(Button);
+        const buttonName = name.charAt(0).toUpperCase() + name.slice(1);
```

Comment:
I think Lodash's [`startCase()`](https://lodash.com/docs#startCase) function could be used here:

```js
import { startCase } from 'lodash';

const buttonName = startCase(name);
```

Reply:
I would normally use that but we are trying to limit the use `lodash` with the idea of removing it as a dependency altogether at some point

Reply:
See above